ed_sensor_integration/src/kinect/fitter.cpp

```diff
@@ -244,10 +230,19 @@ geo::Pose3D computeFittedPose(const geo::Transform2& pose_sensor, ed::EntityCons
 
 // ----------------------------------------------------------------------------------------------------
 
-Fitter::Fitter(uint nr_data_points) :
+Fitter::Fitter()
```

Comment:
As described above. I think we should get rid of this constructor.

Reply:
If we get rid of this we will get errors as the updater class will try to call the default constructor. We also cannot give meaningful values to the parameters of fx or number of beams. without receiving an image.